games/devtest/mods/testnodes/textures.lua

```diff
@@ -255,17 +255,17 @@ data_ck = nil
 
 The following nodes can be used to demonstrate the TGA format support.
 
-Minetest supports TGA types 1, 2, 3 & 10. While adding the support for
+Luanti supports TGA types 1, 2, 3 & 10. While adding the support for
 TGA type 9 (RLE-compressed, color-mapped) is easy, it is not advisable
-to do so, as it is not backwards compatible with any Minetest pre-5.5;
+to do so, as it is not backwards compatible with any Luanti pre-5.5;
```

Comment:
```suggestion
to do so, as it is not backwards compatible with any version pre-5.5;
```
Since it wasn't called Luanti back then